.github/workflows/build-gems.yml

```diff
@@ -100,3 +100,69 @@ jobs:
           path: pkg/*.gem
           if-no-files-found: error
           retention-days: 7
+
+  push:
+    name: Push gems with trusted publishing
+    if: github.event_name == 'release' && github.repository_owner == 'marcoroth'
+    needs: build
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    environment: deployment
```

Comment:
@marcoroth you may want to remove this if you did not configure the env in rubygems